clang/test/C/C2y/n3369.c

```diff
@@ -102,6 +126,11 @@ void test_unspecified_array_length() {
   static_assert(_Countof(**x) == 3);
 }
 
+void test_completed_array() {
+  int a[] = {1, 2, global_num};
+  static_assert(_Countof(a) == 3);
```

Comment:
@AaronBallman This test seems to have caught a bug in the implementation.

I see the following report in the CI:

```
File /var/lib/buildkite-agent/builds/linux-56-59b8f5d88-bt7k7-1/llvm-project/github-pull-requests/clang/test/C/C2y/n3369.c Line 131: invalid application of '_Countof' to an incomplete type 'int[]'
```

But this shouldn't happen.  The type should be completed.  In my GCC implementation, this test passes, AFAIR.

Reply:
Or maybe it's because it was using `errno`, which didn't allow it to be completed.  Maybe it now works.

Reply:
Actually, I don't see it anymore.  Good then.

Reply:
Yeah it was because of the `errno` causing an error.